internal/command/meta_backend_migrate.go

```diff
@@ -790,6 +794,10 @@ func (m *Meta) promptSingleToCloudSingleStateMigration(opts *backendMigrateOpts)
 }
 
 func (m *Meta) promptRemotePrefixToCloudTagsMigration(opts *backendMigrateOpts) error {
+	if !m.input {
+		log.Print("[TRACE] backendMigrateState: can't prompt for input, so aborting migration")
+		return errors.New("error asking for state migration action: input disabled")
```

Comment:
Starting the error message with "error" ends up kind of repetitive, since it gets sprintf'd into a sentence beginning with "Error:"

    │ Error: error asking for state migration action: input disabled

...and likewise the multiple colons. Perhaps something like this? 

```suggestion
		return errors.New("Can't ask approval for state migration when interactive input is disabled")
```

    │ Error: Can't ask approval for state migration when interactive input is disabled

Plus throwing the word "approval" in there helps show the user the real root of the problem, i.e. tf needs to do something that requires permission from a grown-up.

Reply:
Yay!! Thanks for checking through the scenarios @nfagerlund. I agree this message is rough and could be improved. I will update this morning.